src/screens/WebsocketResearchScreen/SignedChatScreen.tsx

```diff
@@ -26,9 +30,17 @@ const SignedChatScreen = () => {
   } = useChatScreenHook(SIGNED);
 
   const flatlistRef = React.useRef<FlatList>();
+  const [loading, setLoading] = React.useState(false);
   const [, dispatchChannel] = (React.useContext(Context) as unknown as any).channel;
   const [profile] = (React.useContext(Context) as unknown as any).profile;
   const updatedChats = updateChatContinuity(chats);
+  const {signedProfileId} = useProfileHook();
+
+  const {moveToAnonymousChannel} = useMoveChatTypeHook();
+
+  const memberChat = selectedChannel?.rawJson?.channel?.members?.find(
+    (item: any) => item.user_id !== signedProfileId
+  );
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The use of `any` type at line 34 and 35 is not recommended as it defeats the purpose of using TypeScript. It's better to define a type or interface for the context. This will improve code maintainability and prevent potential runtime errors.

```diff
- const [, dispatchChannel] = (React.useContext(Context) as unknown as any).channel;
- const [profile] = (React.useContext(Context) as unknown as any).profile;
+ const { channel: [, dispatchChannel], profile: [profile] } = React.useContext(Context);
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->